base/allocator/partition_allocator/BUILD.gn

```diff
@@ -320,6 +330,13 @@ component("partition_alloc") {
     # tagging.cc requires __arm_mte_set_* functions.
     deps += [ "//third_party/android_ndk:cpu_features" ]
   }
+
+  if (is_cobalt && is_starboard && current_toolchain == cobalt_toolchain) {
+    import("//cobalt/build/configs/modular_variables.gni")
+    if (use_custom_libc) {
+      deps += [ "//starboard:starboard_group" ]
```

Comment:
What does the `partition_allocator` actually use from `//starboard_group`?

Reply:
I got a compiler error, lemme dig that up or amybe this is not needed.